services/multiplayer/MultiplayerManager.gd

```diff
@@ -0,0 +1,90 @@
+extends Node
+
+# Maybe we can export theses constants to a global variables project ?
+const DEFAULT_PORT = 28000
+const MAX_PEERS = 4
```

Comment:
Why would you want to export it somewhere else?

Reply:
I understand 👍 
Yet, I think we will later let the host to define them, so we should keep them here.